api/notifications/notifications.go

```diff
@@ -24,22 +22,14 @@ const (
 	LastKnownRevisionHeader     = "last_notification_revision"
 	LastKnownRevisionQueryParam = "last_notification_revision"
 
-	unknownRevision int64 = -1
-	noRevision      int64 = 0
+	noRevision int64 = 0
```

Comment:
Look like this constant is not needed. 
If the revision is Unknown the flow is the same. 

In the producer: https://github.com/Peripli/service-broker-proxy/blob/master/pkg/sbproxy/notifications/producer.go#L43
The value for InvalidRevision is returned when it is unknown.

Reply:
SM sends 0 as last notification when there are no notifications in the database. This 0 is received by the proxy and if its connection drops it should resend it as last known revision. In this case SM sends 410 (see line 47 and 79). Idea to improve this is not to send the 0 at all (both by the SM and proxy). But this requires changes in the proxy as well. It should be done in a different PR.

Reply:
https://github.com/Peripli/service-manager/issues/272